tests/reduction/reduction_common.h

```diff
@@ -0,0 +1,147 @@
+/*******************************************************************************
+//
+//  SYCL 2020 Conformance Test Suite
+//
+//  Provides common code for reduction tests
+//
+*******************************************************************************/
+
+#ifndef __SYCL_CTS_TEST_REDUCTION_COMMON_H
+#define __SYCL_CTS_TEST_REDUCTION_COMMON_H
+
+#include "../common/common.h"
+#include "../common/type_coverage.h"
+
+namespace reduction_common {
+
+static constexpr size_t number_iterations{10};
+
+static sycl::range<1> range{number_iterations};
+static sycl::nd_range<1> nd_range{range, range};
+
+static const auto scalar_types =
+    named_type_pack<char, signed char, unsigned char, short int,
+                    unsigned short int, int, unsigned int, long int,
+                    unsigned long int, float, long long int,
+                    unsigned long long int>(
+        {"char", "signed char", "unsigned char", "short int",
+         "unsigned short int", "int", "unsigned int", "long int",
+         "unsigned long int", "float", "long long int",
+         "unsigned long long int"});
+
+/** @brief Returns expected value for testing
+ *  @tparam VariableT The type of the variable with which the test runs
+ *  @tparam FunctorT The type of the functor with which the test runs
+ *  @tparam BufferT The type of the buffer with which the test runs
+ *  @param functor The functor (plus, multiplies e.t.c) with which the test runs
```

Comment:
```suggestion
 *  @param functor The functor (plus, multiplies, etc.) with which the test runs
```

Reply:
I've applied this comment in this commit: 212a7baae8e03f1861d99eb17467e7b054ca79f4